civis/tests/test_resources.py

```diff
@@ -54,10 +54,8 @@ def test_create_method_iterator_kwarg():
 
 
 def test_create_method_no_iterator_kwarg():
-    # We don't do any validation on keyword arguments if **kwargs is in the
-    # signature. They are just ignored if they aren't in the expected body,
-    # query, or path params. Therefore, method should return a Type Error,
-    # but method2 contains **kwargs so should overwrite iterator = False
+
+    # Test for TypeError when iterator is present and no **kwargs
```

Comment:
I find the statement about kwargs difficult to understand. Perhaps this could say,

> Test that dynamically-created function errors when an unexpected "iterator" parameter is passed in